conf/artemis/broker.xml

```diff
@@ -88,6 +88,7 @@ under the License.
             <global-max-size>100Mb</global-max-size>
 
       -->
+    <global-max-size>2048Mb</global-max-size>
```

Comment:
This might be a little low, is it a random choice? Each sync message is about 750+ bytes in size(take a look in prod), the queue in activeMQ typically can have millions of messages from all remote sites(), it means that 2GB could be small, I would say base the number on the rolling daily total sync messages you receive daily across all sites using stats in prod, you can use that daily report that you run.

Reply:
What amount do you recommend?

Reply:
I did mention in my comment that you will need to use your daily report to see how many messages you get on average in a day from all sites and use that to compute an informed value